src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -484,7 +488,7 @@ private NameSyntax CreateNameSyntax(IReadOnlyList<string> namespaceParts, int in
                 ? usingDirective
                 : usingDirective.WithStaticKeyword(SyntaxFactory.Token(SyntaxKind.StaticKeyword));
 
-            return (usingDirective, addImportService.HasExistingImport(semanticModel.Compilation, root, contextNode, usingDirective));
+            return (usingDirective, addImportService.HasExistingImport(semanticModel.Compilation, root, contextNode, usingDirective, generator));
```

Comment:
don't understand why teh generator would need to be passed in here.

Reply:
Hmm let me double check, I was just passing it down to some place that needed it further down the line.

Reply:
Alright, I believe I tracked it down to the fact that C# and VB generator have not been moved to shared layer. So `VisualBasicAddImportsService.GetGlobalImports` needs the generator for some functionality, but cannot use the VB generator instance directly and does not have access to document to invoke `GetLanguageService<SyntaxGenerator>`.

Reply:
> So VisualBasicAddImportsService.GetGlobalImports needs the generator for some functionality, but cannot use the VB generator instance directly and does not have access to document to invoke GetLanguageService<SyntaxGenerator>

Why cannot it just use VisualBasicSyntaxGenerator.Instance?  It's unclear to me how this could work otherwise.  Either the generaotr is available in this layer, and it should be referencable.  Or it's not available, in which case... how can it be passed in?

Also, with my PR, these types *are* available at this lower layer.  So it seems like we should just go with that.  :)